src/test/util_tests.cpp

```diff
@@ -774,7 +780,7 @@ BOOST_FIXTURE_TEST_CASE(util_ArgsMerge, ArgsMergeTestingSetup)
     // Results file is formatted like:
     //
     //   <input> || <IsArgSet/IsArgNegated/GetArg output> | <GetArgs output> | <GetUnsuitable output>
-    BOOST_CHECK_EQUAL(out_sha_hex, "b835eef5977d69114eb039a976201f8c7121f34fe2b7ea2b73cafb516e5c9dc8");
+    BOOST_CHECK_EQUAL(out_sha_hex, "6e84c6b6a1702f5c2247d175e7e1ad76c342be58b95bf2b7ed7e271d38197a14");
```

Comment:
Are the changes here expected? I don't think they look right. It looks like wallet settings in the top-level of the config file (not in the test section) are now no longer ignored when testnet is enabled.

DIfferences in results.txt before and after this change look like:

```diff
-net=test wallet=c1 wallet=c2 || unset | ignored -wallet
+net=test wallet=c1 wallet=c2 || c1 | c1 c2
...
-net=test wallet=c1 wallet=c2 || unset | ignored -wallet
+net=test wallet=c1 wallet=c2 || c1 | c1 c2
...
```

If this is not intended, it might just be a bug in the test setup, like the `parser.m_network = network;` line above no longer having an effect, instead of a real bug.